strings_lab.js

```diff
@@ -0,0 +1,178 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+
+let firstChar = "macbook air"
+console.log(firstChar[0])
+//
+// 2. `console.log` the length of a string.
+console.log(firstChar.length)
+
+// 3. `console.log` the last character of any string.
+
+console.log(firstChar[firstChar.length-1])
+
+// 4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+let myName = "voniel"
+let lastLetter = myName[myName.length-1]
+//console.log(lastLetter)
+  upperLast = lastLetter.toUpperCase()
+let split = myName.slice(0,5)
```

Comment:
let split = myName.slice(0, myName.length - 1) <====== this would be better code since it can accommodate for differing string length and different name.